libs/analytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java

```diff
@@ -2405,6 +2405,12 @@ public static String getEventNameForStat(AnalyticsTracker.Stat stat) {
                 return "jetpack_powered_banner_tapped";
             case JETPACK_POWERED_BADGE_TAPPED:
                 return "jetpack_powered_badge_tapped";
+            case JETPACK_STATIC_POSTER_DISPLAYED:
+                return "jetpack_static_poster_displayed";
```

Comment:
Hey @ovitrif 👋 
I noticed that the event names are different [from the issues](https://github.com/wordpress-mobile/WordPress-Android/issues/18109) [and iOS](https://github.com/wordpress-mobile/WordPress-iOS/pull/20341/files#diff-50bf5355c5617f11ad12fb2da522fb631bd0db212699e0aa4eb80a5a1ed269caR1255).
Since this is a small change I would suggest to use the `remove_` prefix instead of `jetpack_`. Otherwise give me a heads up to track the new events (ref pcdRpT-24R-p2)

Reply:
Thank you @antonis 👍🏻 , good timing! I'll adapt this to match iOS 🙇🏻

Reply:
Resolved in commit 9d59288

Reply:
Thank you @ovitrif 🙇